File: docs/vs-2015/snippets/csharp/VS_Snippets_VSSDK/loadtoolboxmembers/cs/control1.cs
using System;
using System.ComponentModel;
using System.Drawing;
using System.Windows.Forms;

namespace Company.LoadToolboxMembersCS
{
    //<Snippet03>
    // Set the display name and custom bitmap to use for this item.
    // The build action for the bitmap must be "Embedded Resource".
    [DisplayName("ToolboxMember 1 CS")]
    [Description("Custom toolbox item from package LoadToolboxMembers.")]
    [ToolboxItem(true)]
    [ToolboxBitmap(typeof(Control1), "Control1.bmp")]
    public partial class Control1 : UserControl
    {
        //</Snippet03>

        //<Snippet02>
        public Control1()
        {
            InitializeComponent();

            button1.Text = this.Name + " Button";
        }
        //</Snippet02>

        //<Snippet01>
        private void button1_Click(object sender, EventArgs e)
        {
            MessageBox.Show("Hello world from " + this.ToString());
        }
        //</Snippet01>
    }
}

File: docs/vs-2015/snippets/visualbasic/VS_Snippets_VSSDK/loadtoolboxmembers/vb/control1.vb
'<Snippet03>
' Set the display name and custom bitmap to use for this item.
' The build action for the bitmap must be "Embedded Resource".
<DisplayName("ToolboxMember 1 VB")> _
<Description("Custom toolbox item from package LoadToolboxMembers.")> _
<ToolboxItem(True)> _
<ToolboxBitmap(GetType(Control1), "Control1.bmp")> _
Public Class Control1
    '</Snippet03>

    '<Snippet02>
    Public Sub New()

        InitializeComponent()

        Button1.Text = Me.Name + " Button"
    End Sub
    '</Snippet02>

    '<Snippet01>
    Private Sub Button1_Click(ByVal sender As Object, ByVal e As EventArgs) _
        Handles Button1.Click

        MessageBox.Show("Hello world from " & Me.ToString())
    End Sub
    '</Snippet01>
End Class
